integration-tests/tests/link-to-new-workspace.js

```diff
@@ -1,3 +1,4 @@
+// This test is owned by the Data Explorer Team.
```

Comment:
@petesantos Is `Data Explorer` the official name of the team? I've also seen `Data Catalog`.

Reply:
Yeah, I was wondering if it should be `Data Catalog` given the JIRA Board name is `Data Catalog`.